specification/azsadmin/resource-manager/network/Microsoft.Network.Admin/2015-06-15/Network.json

```diff
@@ -0,0 +1,283 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "description": "Network admin operation endpoints and objects.",
+        "title": "NetworkAdminManagementClient",
+        "version": "2015-06-15"
+    },
+    "host": "adminmanagement.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Network.Admin/adminOverview": {
+            "get": {
+                "x-ms-examples": {
+                    "Get overview of the state of the network resource provider": {
+                        "$ref": "./examples/ResourceProviderState/Get.json"
+                    }
+                },
+                "tags": [
+                    "Network"
+                ],
+                "description": "Get an overview of the state of the network resource provider.",
+                "operationId": "ResourceProviderState_Get",
+                "parameters": [
+                    {
+                        "$ref": "Network.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Network.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/AdminOverview"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Resource": {
+            "description": "Base Resource Object",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "description": "URI of the resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "name": {
+                    "description": "Name of the resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "type": {
+                    "description": "Type of resource.",
+                    "type": "string",
+                    "readOnly": true
+                },
+                "location": {
+                    "description": "Region location of resource.",
+                    "type": "string",
+                    "readOnly": true
```

Comment:
Resource::location is readonly, Quote inherits from Resource, so does this means user cannot specify the location on which the Quote he want to create?

Reply:
The quota endpoints use location based routing, the location property is not used except when set by the RP.